bin/runner.js

```diff
@@ -4,13 +4,12 @@
 const Nightwatch = require('../lib/index.js');
 const {Logger} = require('../lib/utils');
 
-try {
-  Nightwatch.cli(function(argv) {
-    argv._source = argv['_'].slice(0);
+Nightwatch.cli(async function(argv) {
+  argv._source = argv['_'].slice(0);
 
-    const runner = Nightwatch.CliRunner(argv);
-    runner.setup()
-      .startWebDriver()
+  const runner = Nightwatch.CliRunner(argv);
+  await runner.setup().then(() => {
```

Comment:
If the CLI-callback already is async, wouldn't it make sense to refactor this promise-chain as well? (with something like this):

```js
  await runner.setup();
  try {
    try {
      await runner.startWebDriver();
      await runner.runTests();
    } catch (err) {
      runner.processListener.setExitCode(10);
    }
    await runner.stopWebDriver();
  } catch (err) {
    Logger.error(err);
  }
```

Reply:
Yeah that looks way better, thanks!  Will test and push the changes